helix-view/src/view.rs

```diff
@@ -38,18 +38,22 @@ impl JumpList {
         Self { jumps, current: 0 }
     }
 
-    pub fn push(&mut self, jump: Jump) {
+    pub fn push(&mut self, jump: Jump) -> usize {
```

Comment:
Returning the number from `push` is ok but I don't want to let this leak out into the public interface. Let's rename this `push` to `push_impl` and make it private and then introduce a new `pub fn push` that has no return and just calls `self.push_impl(jump);`

Reply:
Sounds good, done